### DISCUSSION TITLE
Fix to record from microphone from linux

### DISCUSSION THREAD

**Discussion by Saikol** [godotengine/godot-proposals#4119] [Engine Core]
Currently the microphone recording system is not functional from linux and android I tried to make a copy of pou but I realized that the audio recording system was broken for these systems I know it is not very necessary but I think it would be good to know will fix.

**Comment by theraot** [comment]
There are bug reports: https://github.com/godotengine/godot/issues/33184 and https://github.com/godotengine/godot/issues/38003

**Comment by Calinou** [comment]
Locking, as this is actually a bug and not a missing feature (see above).
